src/Encamina.Enmarcha.SemanticKernel.Abstractions/Events/MemoryStorageEventEnum.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.ComponentModel;
+
+namespace Encamina.Enmarcha.SemanticKernel.Abstractions.Events;
+
+/// <summary>
+/// <see cref="IMemoryManager.MemoryStorageEvent"/> types.
+/// </summary>
+public enum MemoryStorageEventEnum
```

Comment:
Drop the `Enum` suffix and pluralize the name. It should be `MemoryStorageEvents`.

Reply:
This name is changed to MemoryStorageEventTypes